.github/workflows/python-test.yaml

```diff
@@ -0,0 +1,209 @@
+name: Python Test
+
+on:
+  workflow_run:
+    workflows: [Python Build]
+    types:
+      - completed
+
+permissions:
+  contents: read
+
+jobs:
+  linux:
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          - runner: ubuntu-latest
+            target: x86_64
+          - runner: ubuntu-latest
+            target: x86
+          - runner: ubuntu-latest
+            target: aarch64
+          - runner: ubuntu-latest
+            target: armv7
+        nox-session:
+          # To speed things up a bit we use the special ci_checks_max session
+          # that uses the same venv to run multiple linting sessions
+          - "ci_checks_max"
+          - "pytest_min"
+        python:
+          - "3.11"
+          - "3.12"
+          - "3.13"
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Download wheels
+        uses: actions/download-artifact@v4
+        with:
+          name: wheels-linux-${{ matrix.platform.target }}-${{ matrix.python }}
+          path: dist
+      - name: Create nox venv
+        env:
+          NOX_SESSION: ${{ matrix.nox-session }}
+        run: nox --install-only -e "$NOX_SESSION"
```

Comment:
I think we have a problem with this, because `nox` will still do a `pip install .[dev]` in this step, which means the project will be re-built AFAIK. Did you try this test workflow in your fork?

Reply:
I am using the nox action workflow now, because it works with Maturin.

Reply:
yes, that's why I removed the build pipeline for now